cfpb/consumerfinance.gov
# Remove `wrapper` from `wrapper content_wrapper` pairing

In the design-system we set `content_wrapper` to extend `wrapper` ([here](https://github.com/cfpb/design-system/blob/main/packages/cfpb-layout/src/cfpb-layout.less#L225)), but we have lots of places where we have `wrapper content_wrapper` set together, yet they should be applying the same styles! This PR removes the `wrapper` in that case. 

## Changes

- Remove `wrapper` from `wrapper content_wrapper` pairing


## How to test this PR

1. Some major pages to check for changes (if changes happened they would be related to the padding on the left and right of the page):
 - http://localhost:8000/find-a-housing-counselor/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/
 - http://localhost:8000/data-research/prepaid-accounts/search-agreements/
 - http://localhost:8000/rural-or-underserved-tool/
 - http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/survey/3-5/results/
